api/app/scripts/download_bisq2_media_wiki.py

```
@@ -0,0 +1,389 @@
+    bisq2_root = ET.Element('mediawiki', attrib={
+        'xmlns': "http://www.mediawiki.org/xml/export-0.10/",
+        'version': "0.10",
+        'xml:lang': "en"
+    })
+
+    # Copy the siteinfo section
+    bisq2_root.append(siteinfo)
+
+    # Add only Bisq 2 and relevant general pages
+    bisq2_only_count = 0
+    for title, page_data in all_pages.items():
```

Ruff (0.11.9)

Loop control variable `title` not used within loop body

(B007)